Packs/Vectra_AI/README.md

```diff
@@ -5,26 +5,30 @@ Learn more at [Vectra Website](https://www.vectra.ai).
 
 This pack is designed to quickly integrate with Vectra Detect platform to detect and analyze malicious attacks in progress by creating incident based on Accounts, Hosts or Detections. It gives security engineers visibility into advanced threats to speed detection and remediation response times.
 
+<~XSIAM>This pack includes Cortex XSIAM content.</~XSIAM>
```

Comment:
I'm confused.  Is this an XSOAR pack or an XSIAM pack?  I assume it is an XSIAM pack since it is an Event Collector.  If that's the case, then why do you need to tag this as XSIAM?

Reply:
Because we don't want to display this sentence in XSOAR. It's only relevant for XSIAM.